### DISCUSSION TITLE
Add more accessors to `CompatibleAnimation`

### DISCUSSION THREAD

**Discussion by Lancelotbronner** [airbnb/lottie-ios#2450] [General]
## Which Version of Lottie are you using?

Lottie 4.4.3

## Expected Behavior 

I need to access animation properties, like width and height, from a `CompatibleAnimationView`.
I also load it from data so `CompatibleAnimation` is always nil.

I need a way to access these properties, the more logical place would be `CompatibleAnimation` but that would require refactoring.

## Actual Behavior

These properties are unreachable, `CompatibleAnimationView` has a frame of zero after initialization.
I don't have any way of knowing the size of my animation and our legacy engine needs it.

## Animation JSON
N/A

**Comment by Lancelotbronner** [comment]
My suggestion on [`Sources/Public/iOS/Compatibility/CompatibleAnimationView`](https://github.com/airbnb/lottie-ios/blob/12d01a831acbca68e822f07d035561a02c4349e2/Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift#L203), add the following:

```
@objc var animationWidth: CGFloat { animation?.width ?? 0 }
@objc var animationHeight: CGFloat { animation?.height ?? 0 }
```

**Comment by calda** [comment]
PRs are welcome!
